bindings/typescript/src/ballot.ts

```diff
@@ -0,0 +1,135 @@
+import { ElementModQ } from "./group";
+import {
+  CiphertextBallotHandle,
+  PlaintextBallotHandle,
+  getInstance,
+} from "./wasm";
+
+/**
+ * A PlaintextBallot represents a voters selections for a given ballot and ballot style.
+ *
+ * This class can be either a partial or a complete representation of the expected values of a ballot dataset.
+ * Specifically, a partial representation must include at a minimum the "affirmative" selections
+ * of every contest where a selection is made.  A partial representation may exclude contests for which
+ * no selection is made.
+ *
+ * A complete representation of a ballot must include both affirmative and negative selections of
+ * every contest, AND the placeholder selections necessary to satisfy the NIZKPs for each contest and selection.
+ */
+export class PlaintextBallot {
+  _handle: PlaintextBallotHandle;
```

Comment:
similar to how c# works, there is a native facade and then wrapper classes that can expose other functionality. this is the wrapper class.